src/App.jsx

```diff
@@ -1,104 +1,26 @@
 import './App.scss';
+import { PostList } from './components/PostList/PostList';
```

Comment:
You don`t have to write PostList twice, because all exports from index.js).
And this duplicating repeats in each import.